modules/test/helpers/keystone.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2023 Red Hat
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helpers
+
+import (
+	"github.com/google/uuid"
+	t "github.com/onsi/gomega"
+	keystonev1 "github.com/openstack-k8s-operators/keystone-operator/api/v1beta1"
+	"github.com/openstack-k8s-operators/lib-common/modules/common/condition"
+	k8s_errors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+// CreateKeystoneAPI -
+func (tc *TestHelper) CreateKeystoneAPI(namespace string) types.NamespacedName {
```

Comment:
@stuggi I vaguely remember that we talked about moving this to Keystone itself. But now that I played with the idea I think that is a complex change. That would need a new module in keystone-operator like `keystone-operator/test` so that nova-operator can import that without importing the whole keystone implementation. We could put it to keystone-operator/api but that feels like mixing things into the api module that is not really an api. So I think it is good that we move this here to lib-common. What do you think?

Reply:
@stuggi PTAL

Reply:
sorry, I totally missed this in the mass of emails. I try to do better :) 
Right, we talked about this, but agree that its better to have this in the the helpers here in lib-common.